cli/runScript.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+set -eux
+
+LIMIT_LEDGER_SIZE=500000000
+
+NOOP_PROGRAM_ID="noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV"
+MERKLE_TREE_PROGRAM_ID="JA5cjkRJ1euVi9xLWsCJVzsRzEkT8vcC4rqw9sVAo5d6"
+VERIFIER_PROGRAM_ZERO_ID="J1RRetZ4ujphU75LP8RadjXMf3sA12yC2R44CF7PmU7i"
+VERIFIER_PROGRAM_STORAGE_ID="DJpbogMSrK94E1zvvJydtkqoE4sknuzmMRoutd6B7TKj"
+VERIFIER_PROGRAM_ONE_ID="J85SuNBBsba7FQS66BiBCQjiQrQTif7v249zL2ffmRZc"
+
+solana config set --url http://localhost:8899
+
+if [ -f /.dockerenv ]; then
+    solana-test-validator \
+        --reset \
+        --limit-ledger-size=$LIMIT_LEDGER_SIZE \
+        --faucet-port 9002 \
+        --rpc-port 8899 \
+        --quiet \
+        --bpf-program $NOOP_PROGRAM_ID ~/.local/light-protocol/lib/solana-program-library/spl_noop.so \
+        --bpf-program $MERKLE_TREE_PROGRAM_ID ../light-system-programs/target/deploy/merkle_tree_program.so \
+        --bpf-program $VERIFIER_PROGRAM_ZERO_ID ../light-system-programs/target/deploy/verifier_program_zero.so \
+        --bpf-program $VERIFIER_PROGRAM_STORAGE_ID ../light-system-programs/target/deploy/verifier_program_storage.so \
+        --bpf-program $VERIFIER_PROGRAM_ONE_ID ../light-system-programs/target/deploy/verifier_program_one.so \
+        --account-dir ../../test-env/accounts \
+        &
+    PID=$!
+    trap "kill $PID" EXIT
+
+    sleep 7
+else
+    docker rm -f solana-validator || true
+    docker run -d \
```

Comment:
I have already done this. There is a light test-validator command, which replaces the runTest.sh scripts.

Reply:
Yeah, I meant that for the `light test-validator` command we could also consider running without Docker. We got rid of it in the first place, because of performance issues on macOS.

Reply:
it doesn't use docker anymore but the local solana cli installation

Reply:
I overlooked this file, it's not needed